src/constants.ts

```diff
@@ -1,3 +1,18 @@
+import {
+  API_KEY_SECRET_ARN_ENV_VAR,
+  AWS_LAMBDA_EXEC_WRAPPER_VAR,
+  DOTNET_TRACER_HOME_ENV_VAR,
+  ENVIRONMENT_ENV_VAR,
+  EXTRA_TAGS_ENV_VAR,
+  KMS_API_KEY_ENV_VAR,
+  LAMBDA_HANDLER_ENV_VAR,
+  LOG_LEVEL_ENV_VAR,
+  PROFILER_ENV_VAR,
+  PROFILER_PATH_ENV_VAR,
+  SERVICE_ENV_VAR,
+  VERSION_ENV_VAR,
+} from './commands/lambda/constants'
```

Comment:
Why are these imported here but not used in the file?

Reply:
Good catch, silly mistake by me.

Reply:
Happens to the best of us!